### PR TITLE
DIT-856 fix broken 'attach' link

### PR DESCRIPTION
**Pull request by retroredge**

### CODE REVIEW

File: test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/ReopenCaseViewSpec.scala [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import org.scalatest.BeforeAndAfterEach
+import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes
+import uk.gov.hmrc.tariffclassificationfrontend.models.Operator
+import uk.gov.hmrc.tariffclassificationfrontend.views.ViewMatchers._
+import uk.gov.hmrc.tariffclassificationfrontend.views.html.reopen_case
+import uk.gov.tariffclassificationfrontend.utils.Cases._
+
+class ReopenCaseViewSpec extends ViewSpec with BeforeAndAfterEach {
+
+  override protected def afterEach(): Unit = {
```
Comment (gokyo) [inline test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/ReopenCaseViewSpec.scala]: this method is useless, as you are just calling `super`

File: test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/ReopenCaseViewSpec.scala [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+package uk.gov.hmrc.tariffclassificationfrontend.views
+
+import org.scalatest.BeforeAndAfterEach
+import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes
+import uk.gov.hmrc.tariffclassificationfrontend.models.Operator
+import uk.gov.hmrc.tariffclassificationfrontend.views.ViewMatchers._
+import uk.gov.hmrc.tariffclassificationfrontend.views.html.reopen_case
+import uk.gov.tariffclassificationfrontend.utils.Cases._
+
+class ReopenCaseViewSpec extends ViewSpec with BeforeAndAfterEach {
```
Comment (gokyo) [inline test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/ReopenCaseViewSpec.scala]: is `BeforeAndAfterEach` really needed?

File: test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/ReopenCaseViewSpec.scala (hunk 1 above)
Comment (retroredge) [reply test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/ReopenCaseViewSpec.scala]: copied from an existing test :)

File: test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/ReopenCaseViewSpec.scala (hunk 2 above)
Comment (retroredge) [reply test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/ReopenCaseViewSpec.scala]: copied from an existing test :)

File: test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/ReopenCaseViewSpec.scala (hunk 1 above)
Comment (retroredge) [reply test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/ReopenCaseViewSpec.scala]: Fixed

File: test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/ReopenCaseViewSpec.scala (hunk 2 above)
Comment (retroredge) [reply test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/ReopenCaseViewSpec.scala]: Fixed

**Comment by gokyo** [comment]
LGTM 👍

**Pull request merged**